apollo-compiler/src/test/graphql/com/example/input_object_type/ColorInputExpected.java

```diff
@@ -0,0 +1,57 @@
+package com.example.input_object_type;
+
+import java.lang.Double;
+import javax.annotation.Nullable;
+
+public final class ColorInput {
+  int red = 1;
+
+  @Nullable Double green = 0.0;
+
+  double blue = 1.5;
+
+  ColorInput() {
+  }
+
+  public int red() {
+    return this.red;
+  }
+
+  public @Nullable Double green() {
+    return this.green;
+  }
+
+  public double blue() {
+    return this.blue;
+  }
+
+  public static Builder builder() {
+    return new Builder();
+  }
+
+  public static final class Builder {
+    private final ColorInput colorInput = new ColorInput();
+
+    Builder() {
```

Comment:
well, we can but that not efficient as it will add additional (access$) method to class how is trying to get access to this constructor. It's recommended to have package protected scope for nested classes than private

https://realm.io/news/360andev-jake-wharton-java-hidden-costs-android/